packages/server-wallet/docs/responsibilities.md

```diff
@@ -0,0 +1,41 @@
+## More detail on responsibilities
+### 1. Do not lose secrets
+Store them in a database. 
+
+### 2. Do not leak secrets
+Minimise time spent in memory.
+
+### 3. Store relevant states
+Accept states from counterparties through the `pushMessage` API method.
+In the extreme (“ideal” case), keep every state ever signed or received.
+In practice, garbage collect "old" states (should only need ~`nParticipants` states per channel), by tracking `validTransition` relationships between them. 
```

Comment:
```suggestion
Accept states from counterparties through wire messages.
In the extreme (“ideal” case), keep every state ever signed or received.
In practice, garbage collect "old" states (should only need ~`nParticipants` states per channel), by tracking `validTransition` relationships between them. 
```

It might be nice to comment on why the garbage collecting is safe.